### PR TITLE
🍦Use strict mode for element

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

File: el/templates/element.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
   </template>
   <script>
   (function() {
+    'use strict';
```
Comment (sindresorhus) [inline el/templates/element.html]: I would add a empty line after this.

Comment (samccone) [inline el/templates/element.html]: kk

**Comment by robdodson** [comment]
LGTM, do you mind rebasing into a single commit?

**Comment by samccone** [comment]
I think it is a single commit?
https://github.com/yeoman/generator-polymer/pull/215/commits

:ice_cream:

**Comment by samccone** [comment]
:+1: thanks @robdodson

**Comment by robdodson** [comment]
thank *you*!

**Pull request merged**